### PR TITLE
TST: No conda for OSX and Windows

### PR DESCRIPTION
**Pull request by pllim**
### Description

This pull request is a special backport of #10860 to LTS. 🤞

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -247,7 +247,11 @@ before_install:
 install:
     - if [[ $USE_CI_HELPERS == True ]]; then
         git clone git://github.com/astropy/ci-helpers.git;
-        source ci-helpers/travis/setup_conda.sh;
+        if [[ $TRAVIS_OS_NAME == osx || $TRAVIS_OS_NAME == windows ]]; then
+          source ci-helpers/travis/setup_python.sh;
```
Comment (astrofrog) [inline .travis.yml:251]: This won't actually install PIP_DEPENDENCIES so might need to add a ``pip install $PIP_DEPENDENCIES`` below?

**Comment by bsipocz** [comment]
The whole LTS branch is very badly broken atm, so we can merge this once the windows test passes.

File: .travis.yml (hunk 1 above)
Comment (pllim) [reply .travis.yml:251]: Done. Thanks!

**Comment by pllim** [comment]
Hmm... `WARNING: The wheel package is not available.`

**Comment by pllim** [comment]
Grrr how do I cancel Azure?

**Comment by bsipocz** [comment]
> Grrr how do I cancel Azure?

I suppose we can only let them run, I certainly cannot cancel/rerun those.

File: astropy/_version.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# coding: utf-8
```
Comment (bsipocz) [inline astropy/_version.py:1]: This should not be added

Comment (pllim) [reply astropy/_version.py:1]: Argh, good catch. I had that file from `master`.

**Comment by pllim** [comment]
I don't think the 3 failures are related... The point is the tests are running on Windows.

**Comment by pllim** [comment]
Also, if you think my attempt to fix the imports will mess with backport script, I can back them out. I just wanted to see if the tests would run. Please let me know. Thanks!

**Comment by bsipocz** [comment]
Yes, I agree that this is good to go now. There are issues, but there are also more PRs to backport, limiting this one to backport only the travis config is actually a good thing for smoother backports. (But then there are conflicts now in almost all of them anyway, fortunatelly usually those are still quite simple.)

**Review by bsipocz — APPROVED** [review]

**Comment by bsipocz** [comment]
Thank you @pllim!

**Pull request merged**